app/states/texas/texas_downloader.py

```
@@ -7,90 +9,271 @@
-                for file in files:
-                    ic(f"Removing {file}")
-                    file.unlink()
-
-        driver = webdriver.Chrome(options=options)
-        wait = WebDriverWait(driver, 10)
+            cls,
+            read_from_temp: bool = True,
+            headless: bool = True
+    ) -> Self:
+        tmp = cls.config.TEMP_FOLDER
+        _existing_files = itertools.chain(*map(lambda x: tmp.glob(x), ["*.csv", "*.parquet", "*.txt"]))
```

Ruff (0.14.7)

Unnecessary `map()` usage (rewrite using a generator expression)

Replace `map()` with a generator expression

(C417)